### PR TITLE
Add get_boolean_v and get_float_v

### PR DESCRIPTION
**Pull request by dzbarsky**
[Review on Reviewable](https://reviewable.io/reviews/servo/gleam/53)

### CODE REVIEW

File: src/gl.rs [hunk 1]
```diff
@@ -433,6 +433,20 @@ pub fn get_integer_v(name: GLenum, data: &mut GLint) {
...
+pub fn get_boolean_v(name: GLenum, data: &mut GLboolean) {
+    unsafe {
+        ffi::GetBooleanv(name, data);
+    }
+}
+
+#[inline]
+pub fn get_float_v(name: GLenum, data: &mut GLfloat) {
+    unsafe {
+        ffi::GetFloatv(name, data);
```
Comment (ecoal95) [inline src/gl.rs]: Just for consistency's sake, I'd make these functions return the GLvalue itself (just like `get_buffer_parameter_iv` et al.).

Apart from this I have no more complains, I hope @glennw can review it soon.

**Comment by glennw** [comment]
Yep, agreed. Otherwise looks good to me - sorry I missed this one.

File: src/gl.rs [hunk 2]
```diff
@@ -426,10 +426,31 @@ pub fn tex_sub_image_3d(target: GLenum,
 }
 
 #[inline]
-pub fn get_integer_v(name: GLenum, data: &mut GLint) {
+pub fn get_integer_v(name: GLenum) -> GLint {
```
Comment (dzbarsky) [inline src/gl.rs]: Changed this one also, to be consistent. Do we need to bump to 0.2.0 though?

Comment (glennw) [inline src/gl.rs]: Yes, I believe so.

**Comment by glennw** [comment]
@dzbarsky Looks good to me once conflicts are resolved.

**Comment by dzbarsky** [comment]
(Had to move branches to resolve conflicts)

**Pull request closed without merging**